pages/availability/event/[type].tsx

```diff
@@ -35,6 +36,17 @@ export default function EventType(props) {
     const [ locations, setLocations ] = useState(props.eventType.locations || []);
     const [ customInputs, setCustomInputs ] = useState<EventTypeCustomInput[]>(props.eventType.customInputs.sort((a, b) => a.id - b.id) || []);
     const locationOptions = props.locationOptions
+    const [ dates, setDates] = useState({ startDate: props.eventType.startDate !== null ? dayjs(new Date(props.eventType.startDate)) : dayjs(new Date()), endDate: props.eventType.endDate !== null ? dayjs(new Date(props.eventType.endDate)) : dayjs(new Date()) });
+    const [ timeRange, setTimeRange ] = useState<'unlimited' | 'specific' | undefined>();
+
+    useEffect(() => {
+        if (props.eventType.startDate !== null) {
+            setTimeRange('specific');
+        }
+        else {
+            setTimeRange('unlimited');
+        }
+    }, [])
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has a missing dependency: 'props.eventType.startDate'. Either include it or remove the dependency array.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689307)
```suggestion
    }, [props.eventType.startDate])
```